sample-java-model/src/main/java/com/vimeo/sample_java_model/SwappableParserExampleModel.java

```diff
@@ -0,0 +1,28 @@
+package com.vimeo.sample_java_model;
+
+import com.vimeo.stag.UseStag;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A model to test the case where the consumer
+ * might try to use different parsers for
+ * an object across different gson instances.
+ */
+@UseStag
+public class SwappableParserExampleModel {
+
+    @NotNull
```

Comment:
There are lint warning for the `@NotNull` annotations that they must be initialized.